app/index.html

```diff
@@ -144,6 +146,8 @@
     <script src="bower_components/angular-highlightjs/build/angular-highlightjs.js"></script>
     <script src="bower_components/ng-file-upload/angular-file-upload.js"></script>
     <script src="bower_components/ng-file-upload-shim/angular-file-upload-shim.js"></script>
+    <script src="bower_components/eonasdan-bootstrap-datetimepicker/build/js/bootstrap-datetimepicker.min.js"></script>
```

Comment:
Hmm, I see what you mean. I thought they were similarly styled but I see its pretty different. I agree then, we should change them all, I'm sure we can disable the time component of the new one. It's pretty configurable. I do think this should be done in a separate issue thought. It doesn't fit here. We could create an issue and make sure it gets done before the next release? How does that sound?